### PR TITLE
Rename requested beam parameters

### LINKED ISSUE #29: Distinguish between LFE and KFE
Both lines use the same diagnostic tool, but an instance might be launched for the L line or the K line. Right now it's not immediately obvious which line (KFE or LFE) the diagnostic is pointing to.

Need something to distinguish between the two lines.

### LINKED ISSUE #33: Need to rename Requested Beam Parameters panel to Beam Parameter Status
image

Should also rename "Requested" column to "Limit"

### PR DESCRIPTION
**Pull request by cristinasewell**
Trying to close #33 & close #29 - since they are both little ones I put them together. 

Here is what the Line indicator looks like now, but we can move this anywhere where we find it to be more appropriate - maybe in the description of the PyDM window itself? I'm open for suggestions here, also I added some color to it but if we find it distracting I can remove that, same thing with the additional str: `Line: ` I can get rid of it and only have `KFE` or `LFE` displayed.

line_2

### CODE REVIEW

File: pmps.py [hunk 1]
```diff
@@ -48,6 +47,8 @@ def __init__(self, parent=None, args=None, macros=None):
                 continue
             macros[m] = config.get(m)
 
+        # add CFG macro to display the Line when starting without macros
+        macros['CFG'] = config_name
```
Comment (cristinasewell) [inline pmps.py:51]: Hopefully this won't interfere with anything else?

Comment (ZLLentz) [reply pmps.py:51]: This probably won't interfere with anything

**Comment by ZLLentz** [comment]
Before looking at the diff, I think this looks good visually.

> Here is what the Line indicator looks like now, but we can move this anywhere where we find it to be more appropriate - maybe in the description of the PyDM window itself

Maybe it's valuable to have some redundancy here? Something like "KFE PMPS Diagnostic" in the description as well as the text you've added in the body?

> I added some color to it but if we find it distracting I can remove that, same thing with the additional str: Line: I can get rid of it and only have KFE or LFE displayed.

I think the color and the Line: prefix and the other changes are all very good

**Review by ZLLentz — APPROVED** [review]

**Comment by cristinasewell** [comment]
> Maybe it's valuable to have some redundancy here? Something like "KFE PMPS Diagnostic" in the description as well as the text you've added in the body?

Cool - I'll add that in the description as well

**Comment by cristinasewell** [comment]
Now looks like this:
line_3

**Comment by ZLLentz** [comment]
Awesome! I like it! Ready for merge?

**Comment by cristinasewell** [comment]
> Awesome! I like it! Ready for merge?
cool, yes I think it is ready to merge.

**Pull request merged**